tox.ini

```diff
@@ -29,7 +29,6 @@ commands =
 
     # Override pipenv version of Django as needed by test envs. Use pip so that
     # Pipfile.lock doesn't end up changed by this process.
-    django19: pip install Django~=1.9.0
     django110: pip install Django~=1.10.0
```

Comment:
We should be able to just remove this line, as for now we're installing the correct version with the Pipfile, so no need to "correct" it with this manual installation. May need to purge your old tox environments when running the tests.

Reply:
Also, can we go ahead and improve the indentation of this file? I'm absolutely not a fan of having the variables under an environment start at the same indentation level as the environment name.

Reply:
If we delete it, could the Pipfile.lock change to a different bug fix release or something which would change the version and also possibly dependency versions?

Reply:
I don't think any changes to tox.ini would change Pipfile.lock

Reply:
Oh wait, you are saying the tests will use the lock file anyway, I misunderstood.